src/IntaRNA/InteractionEnergyBasePair.cpp

```diff
@@ -3,5 +3,63 @@
 
 namespace IntaRNA {
 
+void InteractionEnergyBasePair::computeES(const Accessibility &accS,
+    InteractionEnergyBasePair::E2dMatrix &logQ) {
```

Comment:
since you are not using any information from the accS object but the sequence, you can directly pass the sequence only!